hooks/utils.ts

```diff
@@ -1,36 +1,50 @@
 import { Credentials } from "@verida/verifiable-credentials";
-import { Schema, ZkPassResult } from "../@types";
+import { ReclaimResult, Schema, ZkPassResult } from "../@types";
 import { Context } from "@verida/client-ts";
-export const generateVerifiableCredentials = async (context: Context, veridaDid: string, data: ZkPassResult, schema: Schema) => {
+export const generateVerifiableCredentials = async (
+  context: Context,
+  veridaDid: string,
+  data: ZkPassResult | ReclaimResult,
+  schema: Schema
+) => {
   // Note: `context` should already be obtained by connecting to the Verida Network
   const credentialSDK = new Credentials();
 
   // The Verida DID that is the subject of this credential (who is being verified with this credential?)
   // const veridaDid = 'did:vda:testnet:0x7c9699514D3e150e6F50d63345D4bD7B9831da7f';
 
-  const credentialSchema =
+  const zkPassCredentialSchema =
```

Comment:
Maybe put this in the `providers` config file, using an array keyed by `src`?

Reply:
Done